admin/base/views.py

```diff
@@ -1,5 +1,10 @@
-from django.http import HttpResponse
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render
 
+#def root(request):
+#   return HttpResponse("Will probably need to put some front end Auth on this.")
 
-def root(request):
-    return HttpResponse("Will probably need to put some front end Auth on this.")
+@login_required
+def home(request):
+    context = {'user': request.user}
+    return render(request, 'home.html', context)
```

Comment:
This could just be:
```python
return render(request, 'home.html', {'user': request.user})
```
Intermediate variables are good for clarity but only when what you're doing isn't clear to begin with.